lib/clerk/utils.rb

```diff
@@ -1,11 +1,46 @@
+require 'base64'
+
 module Clerk
   module Utils
+    PUBLISHABLE_KEY_LIVE_PREFIX = 'pk_live_'.freeze
+    PUBLISHABLE_KEY_TEST_PREFIX = 'pk_test_'.freeze
+
     module_function
     def camelize(term)
       string = term.to_s
       string = string.sub(/^[a-z\d]*/) { match.capitalize }
       string.gsub!(/(?:_|(\/))([a-z\d]*)/) { "#{$1}#{$2.capitalize}" }
       string
     end
+
+    # see: https://github.com/clerkinc/javascript/blob/main/packages/backend/src/util/parsePublishableKey.ts
```

Comment:
:upside_down_face: I would remove this link, it's very likely that it will become obsolete in the future.

Reply:
What do you mean by becoming obsolete? Do you mean the algorithm change or the `src/util/parsePublishableKey.ts` being moved?
I think we should keep a reference to something that indicates the logic of the implementation.

Reply:
I mean the file being moved. I think links in comments are risky in general, but that's just my opinion.

Of course, feel free to ignore this comment.

At the very least we can use this link instead https://github.com/clerkinc/javascript/blob/c11217cf4432aa640af67ab2fd48f4e5bb33ce6d/packages/backend/src/util/parsePublishableKey.ts
which references a specific commit which will never change.